### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -55,10 +55,11 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://downloads.wordpress.org/release/wordpress-6.3.1.zip"
-        sha256 = "8a3c60c390b1e29143ba49292a5e38d8f7bff2ff6b67a3e4988d157f555a0601"
+        url = "https://downloads.wordpress.org/release/wordpress-6.4.1.zip"
+        sha256 = "280d1e4cbbf862ef48b92e28ac495f729536e10b5403c154f8303279c012f69e"
 
     [resources.system_user]
+    allow_email = true
```
Comment (alexAubin) [inline manifest.toml]: eeeh okay but without any actual use of the `mail_pwd` setting anywhere this serves no purpose ? x_X

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
Merging as the change_url issue is already present on master

**Pull request merged**